src/views/Stake/ExternalStakePool.tsx

```diff
@@ -95,32 +66,15 @@ const MobileStakePool = ({ pool, isLoading }: { pool: ExternalPoolwBalance; isLo
             <Trans>Balance</Trans>
           </Typography>
           <Typography>
-            <>{isLoading ? <Skeleton width={30} /> : `${pool.userBalance} LP`}</>
+            <>{pool.userBalance ? <Skeleton width={30} /> : `${pool.userBalance} LP`}</>
```

Comment:
why is this Mobile balance skeleton check different from desktop balance skeleton check? See below desktop comment.

On mobile the balances don't load if you have 0. 
![image](https://user-images.githubusercontent.com/80423742/149819133-2a4ff478-541b-4117-aba5-27fac739cb0d.png)